infra/gp-multena/templates/03-deployment.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "gp-multena.name" . }}-proxy
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "gp-multena.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "gp-multena.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "gp-multena.selectorLabels" . | nindent 8 }}
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- if .Values.serviceAccount.create }}
+      serviceAccountName: {{ include "gp-multena.name" . }}
```

Comment:
der serviceaccount wird unconditionally im rbac.yaml erzeugt. die create  abfrage hier führt nur dazu, dass keine pods gescheduled werden können, wenn das in den values auf "false" gesetzt wird. der SA wird aber dennoch erzeugt. würd die bedingung hier einfach rauswerfen.